models/user/user.go

```diff
@@ -1233,7 +1233,10 @@ func GetUserByOpenID(uri string) (*User, error) {
 // GetAdminUser returns the first administrator
 func GetAdminUser() (*User, error) {
 	var admin User
-	has, err := db.GetEngine(db.DefaultContext).Where("is_admin=?", true).Get(&admin)
+	has, err := db.GetEngine(db.DefaultContext).
+		Where("is_admin=?", true).
+		OrderBy("id asc"). // Reliably get the admin with the lowest ID.
```

Comment:
Is there a possibility that the user with `ID = 1` is not an admin user?
Because if that is not the case, we could shortcut to `Where("id=?", 1)`.

---

EDIT:
Thinking of it, that certainly fails when the initial admin deleted his account.

Reply:
And also if it's skipped on the install page, it could certainly be that no one makes an admin account until later.

Reply:
(Off-topic, but I doubt this code can be called while there is not a single admin on the instance)